azure-resources/Network/virtualWans/recommendations.yaml

```diff
@@ -0,0 +1,147 @@
+
+- description: Migrate non zone redundant Azure Firewalls to Zone Redundant Azure firewalls by utilizing Firewall Manager
+  aprlGuid: f92e5b09-ca8d-488f-bf91-cf761d261da6
+  recommendationTypeId: null
+  recommendationControl: High Availability
+  recommendationImpact: High
+  recommendationResourceType: Microsoft.Network/azureFirewalls
+  recommendationMetadataState: Active
+  longDescription: Secure Hub might have ended deployed with no Availability Zones. Consider re-deploying your Secure Hub using Firewall Manager so Availability Zones can be configured.
+  potentialBenefits: Reliability of Azure Firewall
+  pgVerified: false
+  publishedToLearn: false
+  publishedToAdvisor: false
+  automationAvailable: no/arg
+  tags: null
+  learnMoreLink:
+    - name: Secure your v-Hub using Azure Firewall Manager
+      url: "https://learn.microsoft.com/en-us/azure/firewall-manager/secure-cloud-network"
+
+- description: Enable routing intent to enable Secured Hubs Interconnectivity

```

Comment:
This appears to be more in the area of enabling functionality vs improving resiliency. I am wondering how this would enhance resiliency. In other words, if asymmetrical flows exist, the customer probably is already aware and has taken steps to correct. So what is the benefit in terms of resiliency of turning this on?

Reply:
When the firewall is non zone redundant and the rack goes down, then there's no resiliency. Adding to different Zones will enhance resiliency in case of Rack going down.

Reply:
Thanks - not sure how this showed up but my feedback was around the "Enable routing intent to enable Secured Hubs Interconnectivity". The non zone redundant FW item makes sense, no concern there!

Reply:
Gotcha, my below comment belongs here then

My train of thought here is, enabling the data path across virtual hubs in case of an outage so it doesn't have to traverse through on premises. This would be recommended if:

Customer has two secure hubs
Customer is not using custom Route tables
Customer doesn't have Routing intent enabled.
Let me know your thoughts

Reply:
This is an architectural recommendation, and does not necessarily make VWAN itself more resilient in terms of configuration, so my feeling, based on having this discussion with Rodrigo in the past, is that this guidance would be associated with architecture, not specifically VWAN itself.

Reply:
Seems to me I can get rid of 

- description: Enable routing intent to send private and internet traffic via Azure Firewall or Security Solution deployed in the v-Hub.

For this one below:
- description: Enable routing intent to enable Secured Hubs Interconnectivity

Enabling inter hub connectivity wouldn't add to their resiliency as a direct path for inter spoke connectivity? If you don't think this guidance belongs here then I can remove it as well.

Reply:
Removed the recommendations pertaining to routing intent as it falls into the WAF option